programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -164,14 +197,21 @@ static int run_opts(int argc, char *argv[]) {
     while ((c = getopt_long(argc, argv, "i:v:r:d:n:",
                             run_options, &option_index)) != -1) {
         switch (c) {
-            case 'i':
-                OPTS.config = strdup(optarg);
+            case 'i': {
```

Comment:
what do you think about supporting a directory (and attempt to load all of the .json identity files in the dir).
Or a single file that contains a list of identity files (and other config info).
possible to add an identity without restarting? eventual IPC support, I guess...

Reply:
possibly even having it be `~/.ziti/identities` might be nice. the good thing is that it makes it easy to move your identity 
 for test reasons... which is the bad thing too possibly.  ZDEW works like this and scans the folder for 'new' identities when it starts up

Reply:
directory option is  coming